### PR TITLE
Added the Images and Updated a text for the SBOM Drift

### PR DESCRIPTION
**Pull request by lavakush07**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: Added the images for the SBOM-Drift 
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

**Review by tejakummarikuntla — CHANGES_REQUESTED** [review]
@lavakush07 , we might need to update the content aligning to the changes on drift feature. For now we are good.

Please revert the changes on `yarn.lock`, it's preferred to keep it untouched.

File: yarn.lock
Comment (wei-harness) [inline yarn.lock:1]: @lavakush07 please don't commit .lock files

**Review by wei-harness — CHANGES_REQUESTED** [review]

File: yarn.lock
Comment (lavakush07) [reply yarn.lock:1]: Yeah sure @wei-harness

File: docs/software-supply-chain-assurance/static/SBOM-Drift-Tag.png
Comment (wei-harness) [inline docs/software-supply-chain-assurance/static/SBOM-Drift-Tag.png:1]: @lavakush07 suggest using kebab-case with all lowercase letters for naming static files.
-
@rohanmaharjan100 would you please add a GH action rule for this case?

**Review by wei-harness — APPROVED** [review]

**Comment by lavakush07** [comment]
Facing the issues while adding images. Will be creating a new PR.

**Pull request closed without merging**